File: solve_problem/codeup/1100/1163/fortune_test_2/fortune_test_2.c
#include <stdio.h>

int main(void)
{
    int year, month, day;
    int num;

    scanf("%d %d %d", &year, &month, &day);

    num = (year + month + day) % 1000;
    num = num / 100;

    if (num % 2 == 0)
    {
        printf("대박");
    }
    else
    {
        printf("그럭저럭");
    }

    return 0;
}

File: solve_problem/codeup/1100/1163/fortune_test_2.py

year, month, day = input().split()
year = int(year)
month = int(month)
day = int(day)

num = (year + month + day) % 1000
num = num / 100

if(num % 2 == 0):
    print('대박')
else:
    print('그럭저럭')
